lib/simple-connect-policy.js

```diff
@@ -6,9 +6,10 @@
 // anonymous handshake.
 //
 module.exports = class SimpleConnectPolicy {
-  constructor(login, password) {
+  constructor(login, password, version) {
```

Comment:
Hm, I thought in the simplest case user doesn't need to change the version dynamically and if they do, they will implement their own connect policy with such behavior. But yeah, I forgot about appName in `connect`, so I'll add version there.